src/routes/profile/ProfileView.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div style="background-color: #4a2416; min-height: 100vh">
+    <v-container class="pa-6">
+      <h1 style="color: rgba(var(--v-theme-surface-2))">
+        My Profile
+      </h1>
+      <p style="color: rgba(var(--v-theme-surface-2))">
+        Username: {{ authStore.username }}
+      </p>
+
+      <!-- Discord -->
+      <v-card flat class="pa-4 mt-4" style="background-color: #4a2416; color: rgba(var(--v-theme-surface-2))">
+        <h2>Discord</h2>
+        <div v-if="hasDiscord">
+          <span style="color: #4caf50">Connected as {{ discordUsername }}</span>
+        </div>
+        <div v-else>
+          <span style="color: #ccc">Not connected</span>
+          <DiscordLink />
+        </div>
+      </v-card>
+
+      <!-- Games List -->
+      <v-card
+        flat
+        class="pa-4 mt-4"
+        style="background-color: rgba(var(--v-theme-surface-2)); color: rgba(var(--v-theme-surface-1))"
+      >
+        <h2>My Games</h2>
+
+        <v-virtual-scroll
+          v-if="games.length > 0"
+          :items="games"
+          height="600"
+          item-height="100"
+        >
```

Comment:
I think a table might work better here, especially if we implement pagination.

@itsalaidbacklife penny for your thoughts